apps/cowswap-frontend/src/legacy/components/AddressInputPanel/index.tsx

```diff
@@ -46,7 +46,7 @@ const Input = styled.input<{ error?: boolean }>`
   flex: 1 1 auto;
   background: none;
   transition: color 0.2s ${({ error }) => (error ? 'step-end' : 'step-start')};
-  color: ${({ error }) => (error ? `var(${UI.COLOR_DANGER}` : 'inherit')};
+  color: ${({ error }) => (error ? `var(${UI.COLOR_DANGER})` : 'inherit')};
```

Comment:
😱

Reply:
doesn't our lint tell this things? 

like:
https://github.com/styled-components/stylelint-processor-styled-components